README.md

```diff
@@ -61,6 +141,22 @@ const user = await xata.db.users.create({
 });
 ```
 
+If you want to create a record with a specific ID, you can invoke `insert()`.
+
+```ts
+const user = await xata.db.users.insert('user_admin', {
+  full_name: 'John Smith'
```

Comment:
```suggestion
  fullName: 'John Smith'
```

I know we were not using camel case in other examples, but I mainly did that because of the default schema file that `xata init` generates. However I think it'd better to use camel case in our docs.